assets/wizards/googleAdSense/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Google AdSense Wizard.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, render, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { withWizard, FormattedHeader, Handoff } from '../../components/src';
+import './style.scss';
+
+/**
+ * External dependencies
+ */
+import { HashRouter, Redirect, Route, Switch } from 'react-router-dom';
+
+/**
+ * Subscriptions wizard for managing and setting up subscriptions.
+ */
+class GoogleAdSenseWizard extends Component {
+
+	render() {
+		const { pluginRequirements } = this.props;
+		return (
+			<HashRouter hashType="slash">
+				<Switch>
+					<Fragment>
+						<FormattedHeader
+							headerText={ __( 'Google AdSense' ) }
+							subHeaderText={ __( 'Connect to your AdSense account using the Site Kit plugin, then enable Auto Ads.' ) }
+						/>
+						<Handoff
+							plugin='google-site-kit'
+							editPath='admin.php?page=googlesitekit-module-adsense'
+							className='is-centered'
+							isPrimary
+						>{ __( 'Set up Google AdSense' ) }</Handoff>
+					</Fragment>
+				</Switch>
+			</HashRouter>
+		);
+	}
+
```

Comment:
After completing the set up, I'm dropped back on this page which continues telling me to set up AdSense. The 'Back to checklist' link reads like a 'Cancel' IMO. Would it make things clearer to also have an "I've finished setting up AdSense" button underneath the handoff button? It would take you to the checklist, and (in the future) it would mark the task complete in the checklist.

e.g.
![localhost_wp-admin_admin php_page=newspack-google-adsense-wizard](https://user-images.githubusercontent.com/7317227/60277113-c8fa7000-98b1-11e9-849e-aad4d14d753a.png)

Reply:
That's a good idea. I'm going to make a separate issue of that

Reply:
https://github.com/Automattic/newspack-plugin/issues/135